x-pack/plugins/cloud_security_posture/public/pages/rules/rules_container.tsx

```diff
@@ -93,6 +93,15 @@ export const RulesContainer = () => {
 
   const { mutate: bulkUpdate, isLoading: isUpdating } = useBulkUpdateCspRules();
 
+  const lastModified = useMemo(
+    () =>
+      data?.savedObjects
+        .map((v) => v.updatedAt)
+        .filter(isNonNullable)
+        .sort((a, b) => +new Date(b) - +new Date(a))[0] || '',
```

Comment:
I'm pretty sure sort can sort dates without changing to numbers, give it a check

Reply:
maybe try to yell back